test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -214,7 +214,16 @@
          IllegalArgumentException
          #".* 'delete', 'get', 'head', 'post', 'put'"
          (validate-auth-config-rule!
-          (assoc-in testrule [:match-request :method] ["nasty" "gross"]))))))
+          (assoc-in testrule [:match-request :method] ["nasty" "gross"])))))
+
+    (is (thrown-with-msg?
```

Comment:
Seems like this one shouldn't fail with the "at least one capture group" message.  A regex path isn't required to have capture groups.  In this case, there are no capture groups and none of the ACEs make any backreferences.  So this one looks valid to me.